src/server/journal/executor.cc

```diff
@@ -0,0 +1,34 @@
+// Copyright 2022, DragonflyDB authors.  All rights reserved.
+// See LICENSE for licensing terms.
+//
+
+#include "server/journal/executor.h"
+
+#include "base/logging.h"
+#include "server/main_service.h"
+
+namespace dfly {
+
+JournalExecutor::JournalExecutor(Service* service) : service_{service} {
+}
+
+void JournalExecutor::Execute(journal::ParsedEntry&& entry) {
```

Comment:
I'm not sure if it deserves a whole separate file and what to do with it in the future.

I recently introduced this cut-down serializer_commons. But in the subsequent PR we should probably unite all the parsing stuff further

Reply:
Besides this class will grow in the future and become more complicated to coordinate on transactions